### PR TITLE
feat(docker): add pg-meta

### PR DESCRIPTION
**Pull request by soedirgo**

### CODE REVIEW

**Review by kiwicopple — CHANGES_REQUESTED** [review]

File: docker/volumes/kong.yml [hunk 1]
```diff
@@ -76,10 +132,31 @@ services:
...
     plugins:
       - name: cors
-consumers:
-  - username: anon
-    keyauth_credentials:
-      - key: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJyb2xlIjoiYW5vbiIsImlhdCI6MTYyNzIwODU0MCwiZXhwIjoxOTc0MzYzNzQwfQ.zcaQfHd3VA7XgJmdGfmV86OLVJT9s2MTmSy-e69BpUY
-  - username: service_role
-    keyauth_credentials:
-      - key: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJyb2xlIjoic2VydmljZV9yb2xlIiwiaWF0IjoxNjI3MjA4NTQwLCJleHAiOjE5NzQzNjM3NDB9.pkT3PNpO4DtO45Ac5HK_TKCx8sGLgNtV__pr_ZrRSAU
+      - name: key-auth
```
Comment (awalias) [inline docker/volumes/kong.yml:135]: this might have been for publicly available files and buckets (images etc.)? @inian should be able to verify

Comment (inian) [reply docker/volumes/kong.yml:135]: yeah, storage doesn't need key-auth from kong. it manages its own authentication

**Review by kiwicopple — APPROVED** [review]

File: docker/volumes/kong.yml (hunk 1 above)
Comment (kiwicopple) [reply docker/volumes/kong.yml:135]: is it still useful to keep a layer of protection in Kong? Block requests at the edge?

Comment (kiwicopple) [reply docker/volumes/kong.yml:135]: we could add in a open route/service for any public storage routes? Up to you - happy to revert if it's completely unnecessary

File: docker/test.http [hunk 2]
```diff
@@ -0,0 +1,22 @@
+## For use with the VS Code Extension "REST Client"
```
Comment (soedirgo) [inline docker/test.http:1]: Is this file necessary? I think this can be omitted.
